scripts/dts/edtlib.py

```diff
@@ -160,12 +161,17 @@ def __init__(self, dts, bindings_dirs, warn_file=None,
           If True, a warning is printed if a node has a 'reg' property where
           the address of the first entry does not match the unit address of the
           node
+
+        default_prop_types (default: False):
+          If True, default property types will be used when a node has no
```

Comment:
Can we default this to True and update the old script to call it with False.

Reply:
> Can we default this to True and update the old script to call it with False.

done